### DISCUSSION TITLE
[deepin 20.1] Window in administrative mode

### DISCUSSION THREAD

**Discussion by lxwndr** [linuxdeepin/developer-center#2939] [Features Request & Ideas | 特性请求 & 头脑风暴]
Which of the windows is in administrative mode?

![Captura de Tela_select-area_20201230145326](https://user-images.githubusercontent.com/45263160/103371835-355dbd00-4aaf-11eb-9ade-bcd99fad871a.png)

<br>

Please differentiate between the windows; so that the user knows whether or not he is in administrative mode.

The administrative window can display a theme with the motherboard tracks (in light gray):

![image](https://user-images.githubusercontent.com/45263160/103372156-07c54380-4ab0-11eb-8623-2f9ecbd7e878.png)

<br>

deepin 20.1 (1010) / OsBuild=11031.108

**Comment by lxwndr** [comment]
The problem still persists in the new deepin 20.2. The users cannot visually differentiate an administrative window from the File Manager, to a normal window, without privileged access.

dde-file-manager 5.2.8-1
deepin 20.2 (1040) / OsBuild=11038.001

**Comment by deepin-CD** [comment]
Thanks for your feedback, we will consider it.

**Comment by meiyixiang** [comment]
I understand what you mean. I have fed back your suggestions to the relevant product manager

**Comment by lxwndr** [comment]
Any news?
